relm4-macros/tests/component-missing-widgets.rs

```diff
@@ -13,9 +13,9 @@ impl SimpleComponent for TestComponent {
     }
 
     fn init(
-        init: Self::Init,
+        _: Self::Init,
         root: &Self::Root,
-        sender: ComponentSender<Self>,
+        _sender: ComponentSender<Self>,
     ) -> ComponentParts<Self> {
```

Comment:
This could also be a `compile-pass` test, maybe we should start a separate folder for those?

Reply:
I also thought about this, but then I realized that every regular test case is already a compile-pass test technically.

Reply:
Personally I would move the regular tests to compile-pass tests then. There we get the additional checking that no warnings are emitted.

Reply:
I'll merge this PR for now, but we can address this in another PR if you want.